Makefile

```diff
@@ -0,0 +1,2 @@
+fastparquet/parquet_thrift: fastparquet/parquet.thrift
```

Comment:
@martindurant not sure what your thoughts are on adding a Makefile. I can never remember where the dashes go the `thrift` commands though :)

Reply:
Does this squash the existing generated files each time?
I am no expert on `make`; but will this work on every platform?

Reply:
> Does this squash the existing generated files each time?

You can run `make` whenever, it'll only do anything when `parquet.thrift` changes. It's identical to just running those commands manually.

>  I am no expert on make; but will this work on every platform?

Linux / mac yes (maybe Mac needs `xcode-select` installed, can't remember).

Windows, yes with some work getting the PATH configured correctly. Even if it doesn't work for contributors, I think having it here is useful just to remember the command.

Reply:
OK, then no objection from me.